server/db/models/Product.js

```diff
@@ -1,12 +1,48 @@
 const { STRING } = require('sequelize')
 const db = require('../db')
 
+const Product = conn.define('product', {
+    title: {
+        type: STRING,
+        allowNull: false,
+        validate: {
+            notEmpty: true,a
```

Comment:
You a random 'a' at the end of this line

Reply:
Done!